rmgpy/data/base.py

```diff
@@ -75,6 +75,8 @@ class Entry(object):
     `metal`             Which metal the thermo calculation was done on (``None`` if not used)
     `facet`             Which facet the thermo calculation was done on (``None`` if not used)
     `site`              Which surface site the molecule prefers (``None`` if not used)
+    `binding_energies'  The surface binding energies for C,H,O, and N
+    `surface_site_density`  The surface site density
```

Comment:
Do any database saving (writing) methods need updating with these new attributes, or did you take care of that?